packages/e2e-tests/specs/block-transforms.test.js

```diff
@@ -159,7 +159,15 @@ maybeDescribe( 'Block transforms', () => {
 			)
 		);
 
-		it.each( testTable )(
+		// Group is an available transform on all Blocks. Testing each transform adds
+		// significant overhead to the tests. Therefore we filter these out and test
+		// Group transforms separately.
+		const testTableWithoutGroup = testTable.filter( ( transform ) => ! transform.includes( 'Group' ) );
```

Comment:
I guess we should remove this assignment and the comment before it.

Reply:
Fixed